### PR TITLE
Math Magician:Components

### PR DESCRIPTION
**Pull request by MarkYegon7**
In this project, I was able to:

- Use correct Gitflow principles.
- Use linters.
- Enable React App to run properly

### CODE REVIEW

File: src/components/calc.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+//       <input type="text" value="0" readOnly />
+//     </div>
+//     <div className="calculator-buttons">
+//       {/* Your calculator buttons go here */}
+//     </div>
+//   </div>
+// );
+
+// export default Calculator;
+
```
Comment (AmaduKamara) [inline src/components/calc.js]: - [ ] It is a best coding practice to not have unused commented code in our codebase as they make our code look dirty. Kindly remove all commented unused codes from this file 👍🏽

Comment (AmaduKamara) [inline src/components/calc.js]: - [ ] It is a best coding practice to not have unused commented code in our codebase as they make our code look dirty. Kindly remove all commented unused codes from this file 👍🏽

File: src/components/calc.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import React from 'react';
+import './calc.css';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="calculator-display">
+      <input type="text" value="0" readOnly />
+    </div>
```
Comment (AmaduKamara) [inline src/components/calc.js:23]: - [ ] According to the project requirements, you're required to have at least one child component being used inside the Calculator.js file. It's for the purpose of practicing the use of props and more components usage. Kindly create at least one child component that is used inside this component and take a prop used inside of it. 👍🏽

Screenshot 2023-09-28 at 5 39 16 PM

## Suggestions:

- [ ] You could create calculator result component and render it in the place of the result section of the design and pass a result value to it as a prop 👍🏽
- [ ] Or, you create Button component that you could use in place of the button element 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @MarkYegon7,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You set up Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ The calculator UI is nicely styled 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/calc.js (hunk 1 above)
Comment (MarkYegon7) [reply src/components/calc.js]: Done

Comment (MarkYegon7) [reply src/components/calc.js]: Done

File: src/components/calc.js (hunk 2 above)
Comment (MarkYegon7) [reply src/components/calc.js:23]: Done

File: src/components/calc.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './calc.css';
+// import Button from './Button';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="calculator-display">
+      <input type="text" value="0" readOnly />
+    </div>
```
Comment (ghost) [inline src/components/calc.js]: - [ ] As requested earlier, the calculator component should have at least one child component.  You created a `button.js` component but you did not use it. Kindly also  avoid leaving commented-out code on your codebase to keep it clean

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @MarkYegon7 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ App is designed as expected 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calc.js (hunk 3 above)
Comment (Abenezer-Tilahun) [reply src/components/calc.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @MarkYegon7 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**